service-impl/audit-service-impl/logger-audit-service/src/main/java/uk/gov/gchq/palisade/audit/service/impl/LoggerAuditService.java

```diff
@@ -58,6 +58,7 @@
     @Override
     public void writeConfiguration(final InitialConfig config) {
```

Comment:
Is it worth renaming this method to be writeDefaultConfiguration. I feel that in a real deployment scenario then the config might get changed and we don't want this to override those changes. I also don't feel particularly comfortable having config being set within the service implementations. I know you have done it this way to reduce code duplication for the examples, however we have to keep in mind that these would be deployed and configured by an admin and not be spin up tear down instances like we have set in the examples.

Reply:
It could be worth renaming the Service.configure and Service.writeConfiguration methods to readConfig and writeConfig or something that signifies they come as a pair. Although currently only used to write the default configurations for examples, these methods are how individual services write and read their configuration to a InitialConfig object. Whenever a service wants to record its configuration for later storage by the config service, the writeConfiguration method needs to be called. Therefore we shouldn't remove it. At the point an admin is configuring a service, at the end the service would be asked to write its configuration.